python/pyspark/sql/tests.py

```diff
@@ -1937,19 +1937,6 @@ def test_list_functions(self):
             className="org.apache.spark.sql.catalyst.expressions.Add",
             isTemporary=True))
         self.assertEquals(functions, functionsDefault)
-        spark.catalog.registerFunction("temp_func", lambda x: str(x))
-        spark.sql("CREATE FUNCTION func1 AS 'org.apache.spark.data.bricks'")
```

Comment:
Is there a reason why we removed creating the function with an missing class name?